controller-client/src/main/java/org/jboss/as/controller/client/logging/ControllerClientLogger.java

```diff
@@ -372,6 +374,10 @@
     @Message(id = 37, value = "Failed to parse the configuration file: %s")
     RuntimeException failedToParseAuthenticationConfig(@Cause Throwable cause, URI location);
 
+    @LogMessage(level = WARN)
+    @Message(id = 38, value = "Property or feature %s not supported by %s")
+    void xmlFactoryPropertyNotSupported(@Cause Throwable cause, String property, String factoryName);
```

Comment:
Is it expected these would never be logged? Otherwise recurring WARNs would be problematic.

Reply:
I apologize but I somehow unnoticed the ongoing Zulip discussion about this PR @boris-unckel .
I agree with both of you @bstansberry and @dmlloyd this kind of change should be in a more generic
module / library like e.g. wildfly-common, because this PR introduces some "optimized XML processing" settings
and these settings might be reused by other modules / libraries / WS subsystems.

Reply:
@boris-unckel Thanks. In the WildFly context, is the expectation that these would never be logged? @ropalka you may know the answer too.

My assumption is we don't expect these to be logged; i.e. the factories we'd wrap will support all of these features. But I don't want to assume. :)